botnim/cli.py

```diff
@@ -53,68 +53,65 @@ def reverse_lines(text: str) -> str:
     return "\n".join(reversed_lines)
 
 @query_group.command(name='search')
-@click.argument('environment', type=click.Choice(['production', 'staging']))
-@click.argument('bot', type=click.Choice(AVAILABLE_BOTS), default='takanon')
+@click.argument('environment', type=click.Choice(VALID_ENVIRONMENTS))
+@click.argument('bot', type=click.STRING)
```

Comment:
why?

Reply:
[and trying to be consistent...]

Reply:
I'll elaborate - why did you change the type of this parameter from `type=click.Choice(AVAILABLE_BOTS)` to `type=click.STRING`?

Reply:
oh, I did not consider well the meaning of this change. fixed.